CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.9.5 - 2024-09-11
+### Fixed
+* [[915]](https://github.com/microsoft/vscode-azureresourcegroups/pull/915) Remove bad implementation of `createSubscriptionContext`
+
+### Engineering
+* [[913]](https://github.com/microsoft/vscode-azureresourcegroups/pull/913) Upgrade packages for additional telemetry updates
+
```

Comment:
Should we prepare this one for the signing pipeline before release?

Reply:
Working on it 🏃💨

Reply:
I'll defer to @alexweininger for input on this one

Reply:
No rush! If you need a couple extra days to test stuff let me know and we can move the release timeline around

Reply:
https://github.com/microsoft/vscode-azureresourcegroups/pull/919

Reply:
Let me know if that changelog update is more user friendly / if you have another way you'd prefer it be worded